.github/workflows/prepare-test-db.yml

```
@@ -57,81 +58,70 @@ jobs:
           docker rm supabase_db_tet  || echo "container already removed"
           docker volume rm supabase_db_tet || echo "volume already removed"
 
-      - name: Start supabase
+      - parallel:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)